docs/machine-learning/tutorials/health-violation-classification-model-builder.md

```diff
@@ -0,0 +1,169 @@
+---
+title: 'Tutorial: Classify health violations with Model Builder'
+description: This tutorial illustrates how to build a multiclass classification model using ML.NET Model Builder to classify restaurant health violation severity in San Francisco.
+author: luisquintanilla
+ms.author: luquinta
+ms.date: 10/29/2019
+ms.topic: tutorial
+ms.custom: mvc
+#Customer intent: As a non-developer, I want to use Model Builder to automatically generate a model to classify violation severity using Model Builder.
+---
+
+# Tutorial: Classify the severity of restaurant health violations with Model Builder
+
+Learn how to build a multiclass classification model using Model Builder to categorize the risk level of restaurant violations found during health inspections.
+
+In this tutorial, you learn how to:
+
+> [!div class="checklist"]
+>
+> - Prepare and understand the data
+> - Choose a scenario
+> - Load data from a database
+> - Train the model
+> - Evaluate the model
+> - Use the model for predictions
+
+> [!NOTE]
+> Model Builder is currently in Preview.
+
+## Prerequisites
+
+For a list of prerequisites and installation instructions, visit the [Model Builder installation guide](../how-to-guides/install-model-builder.md).
+
+## Model Builder multiclass classification overview 
+
+This sample creates a C# .NET Core console application that categorizes the risk of health violations using a machine learning model built with Model Builder. You can find the source code for this tutorial at the [dotnet/machinelearning-samples](https://github.com/dotnet/machinelearning-samples) GitHub repository.
```

Comment:
Will you link to actual sample?

Reply:
Yes. Once it's merged into master.

#resolved